recipes/libs/gdbm.yaml

```diff
@@ -14,6 +14,7 @@ checkoutScript: |
     # prevent Makefile from updating the file because it's older than "configure"
     touch doc/stamp-vti
 
+buildTools: [gettext]
```

Comment:
Hmm, maybe fold this commit into the previous one? After all, there you removed `checkoutTools: [gettext]`...